game/services_avatar.py

```
@@ -0,0 +1,110 @@
+    if not filename or "." not in filename:
+        return ""
+    return filename.rsplit(".", 1)[-1].lower().strip()
+
+
+def _guess_image_format(file_bytes: bytes) -> str:
+    # imghdr is not available in some Python builds.
+    # Fallback: rely on Pillow's decoder for image-type sanity.
+    try:
+        with Image.open(io.BytesIO(file_bytes)) as img:
+            return (img.format or "").lower()
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)